doc/source/conf.py

```diff
@@ -88,6 +106,9 @@
 numpydoc_show_class_members = False
 numpydoc_xref_param_type = True
 
+# autodoc configuration
+autoclass_content = "both"
```

Comment:
@clatapie I planned to include this in my PR, as it relates to the issue I'm trying to fix there (#153).
Maybe it's best to not add it in both PRs to avoid confusion, what do you think? (or does it also affect your other changes here ?)